src/Aspire.Hosting/api/Aspire.Hosting.cs

```diff
@@ -147,6 +153,12 @@ public ApplicationModel.IResourceBuilder<T> CreateResourceBuilder<T>(T resource)
             where T : ApplicationModel.IResource { throw null; }
     }
 
+    public static partial class DistributedApplicationBuilderExtensions
+    {
+        public static ApplicationModel.IResourceBuilder<T> CreateResourceBuilder<T>(this IDistributedApplicationBuilder builder, string name)
```

Comment:
I know there was a little bit of discussion in https://github.com/dotnet/aspire/issues/3022#issuecomment-2628438596, but I kind of think `Get` makes more sense here. Yes it "creates a resource builder", but builders are ephemeral. I think it is a bit confusing to call this "Create" when logically you are just getting a builder wrapped around the existing resource.

Thoughts @davidfowl @mitchdenny @ReubenBond @DamianEdwards ?

Reply:
Hmm, that's a good point but we are creating a builder over an existing resource (this also already exists today for an IResource).

Reply:
ok, I guess we are past the point of feedback. I would have called this `Get`.